sagemaker-clarify/fairness_and_explainability/fairness_and_explainability_boto3.ipynb

```diff
@@ -0,0 +1,1541 @@
+    "  * `**shap:**` This section has the parameter for SHAP analysis. Kernel SHAP algorithm requires a baseline (also known as background dataset). If not provided, a baseline is calculated automatically by SageMaker Clarify using K-means or K-prototypes in the input dataset. Baseline dataset type shall be the same as `dataset_type`, and baseline samples shall only include features. By definition, `baseline` should either be a S3 URI to the baseline dataset file, or an in-place list of samples. In this case we chose the latter, and put the mean of the train dataset to the list. For more details on baseline selection please [refer this documentation](https://docs.aws.amazon.com/en_us/sagemaker/latest/dg/clarify-feature-attribute-shap-baselines.html).\n",
+    "* `predictor` includes model configuration, this section is required if the analysis requires predictions from model\n",
+    "  * `model_name`: name of the concerned model, using name of the xgboost model trained earlier, `DEMO-clarify-xgboost-model`\n",
+    "  * `instance_type` and `initial_instance_count` specify your preferred instance type and instance count used to run your model on during SageMaker Clarify's processing. The testing dataset is small so a single standard instance is good enough to run this example.\n",
+    "  * `accept_type` denotes the endpoint response payload format, and `content_type` denotes the payload format of request to the endpoint. As per the example model we created above both of these will be `text/csv`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO:botocore.credentials:Found credentials from IAM Role: BaseNotebookInstanceEc2InstanceRole\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "'s3://sagemaker-ap-south-1-003394947794/sagemaker/DEMO-sagemaker-clarify-boto3/explainability_analysis_config.json'"
+      ]
+     },
+     "execution_count": 30,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Upload the analysis_config to the concerned S3 path.\n",
+    "S3Uploader.upload(\"analysis_config/explainability_analysis_config.json\", \"s3://{}/{}\".format(bucket, prefix))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Run Sagemaker Clarify Processing Job"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      ".............\n"
+     ]
+    }
+   ],
+   "source": [
+    "processing_job_name = create_processing_job(explainability_analysis_config_path, explainability_analysis_result_path)\n",
+    "wait_for_job(processing_job_name)"
```

Comment:
Good to warn the user how long the job runs. E.g. "The job takes about 10 minutes to run."